lib/datadog/security.rb

```diff
@@ -0,0 +1,13 @@
+require 'datadog/security/configuration'
+
+module Datadog
+  # Namespace for Datadog Security instrumentation
+  module Security
+    include Configuration
+  end
```

Comment:
I don't have much insight into configuration, and I also had some issues trying to make it independent while reusing code, maybe because of the global-ish instance variable you mention (at some point, however I sliced it, there was some common reference that mixed things up, but that might be me lacking knowledge of the implementation). I agree there's some need for sharing the configuration code inside of core, and being able to have proper segregation possible (modulo I could not achieve it myself).

Reply:
Mega fair enough! Maybe let's keep this aside and I'll be on the look out on how to improve this separately :)